apps/web/src/components/sharing/ShareDialog.test.tsx

```
@@ -74,13 +91,17 @@ function sharingEngine(refusals: Record<string, Error> = {}, held: Partial<Engin
           contacts: state.contacts.map((seed) => ({
             identityPublicKey: identity(seed),
           })),
-          grants:
+          state:
```

ESLint

[error] Delete `··`

(prettier/prettier)